add new hometask and add in repository task in lesson

AnnSchiretz opened the pull request:
add some changes in .gitignore

AnnSchiretz committed:
add new hometask and add in repository task in lesson
add some changes in .gitignore

AksPokl commented on homeTask/src/main/java/com/teachmeskill/hometask1.3/NumberMin.java:
```diff
@@ -0,0 +1,15 @@
+public class NumberMin {
```
лучше MinNumberApp

AksPokl commented on homeTask/src/main/java/com/teachmeskill/hometask1.3/RoundingTheNumber.java:
```diff
@@ -0,0 +1,13 @@
+public class RoundingTheNumber {
+    public static void main(String[] args) {
+        double num = 87.859;
+        rouding(num);
```
Метод лучше называть, используя глагол
round();

AksPokl commented on homeTask/src/main/java/com/teachmeskill/hometask2/Compute.java:
```diff
@@ -0,0 +1,18 @@
+public class Compute {
```
А класс называть, используя глагол не надо =)

AksPokl commented on homeTask/src/main/java/com/teachmeskill/hometask2/EvenNumber.java:
```diff
@@ -0,0 +1,13 @@
+                continue;
+            }
+        } while (num <= 100);
+
```
Лишняя строка

AksPokl commented on homeTask/src/main/java/com/teachmeskill/hometask2/Sportsman.java:
```diff
@@ -0,0 +1,11 @@
+public class Sportsman {
+    public static void main(String[] args) {
```
В задаче надо было найти общую дистанцию.

AksPokl reviewed (changes requested):
Summary:
- убрать лишние пустые строки
- подумать над названиями методов и классов
- дорешать задачу

AnnSchiretz committed:
delete empty line
rename methods and classes
finish task Sportsman.java (add total distance)
delete unnecessary folder (homeTask)

AksPokl commented on src/main/java/com/teachmeskill/hometask2/SumDistanceSportmanApp.java:
```diff
@@ -0,0 +1,18 @@
+        double totalAmount = totalDistance(distance, increase);
+        System.out.println(totalAmount);
+    }
+    public static double totalDistance (double distance, double increase) {
```
getTotalDistance ();
countTotalDistance ();

AnnSchiretz committed:
rename method in SumDistanceSportmanApp.java
